enterprise_access/apps/api_client/ecommerce_client.py

```diff
@@ -0,0 +1,47 @@
+"""
+API client for calls to the ecommerce service.
+"""
+from django.conf import settings
+
+from enterprise_access.apps.api_client.base_oauth import BaseOAuthClient
+
+
+class EcommerceApiClient(BaseOAuthClient):
```

Comment:
Same question for the `LicenseManagerAPIClient`: Can we set a client side timeout of these requests to ~45 seconds? This way this code base can catch the timeout and do something sane without instead of leaving the users browser spinning or returning a generic 500 error?

Reply:
Added in 45 second timeouts for both clients